std_msgs/Header header
pose_classifier/PoseClassification[] poses
